addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -1113,6 +1113,9 @@ export class PosOrder extends Base {
     getName() {
         return this.getFloatingOrderName() || "";
     }
+    isUnsyncedPaidOrder() {
```

Comment:
Maybe do a getter `isUnsyncedPaid` don't need to put `order` in the name since it will be call from an order object.